tests/dialects/test_affine.py

```diff
@@ -77,3 +81,21 @@ def test_for_mismatch_blockargs():
 def test_yield():
     yield_ = YieldOp.get()
     assert yield_.arguments == ()
+
+
+def test_affine_apply_map():
+    MODULE_CTX = """
+#map = affine_map<()[s0] -> (s0 * 4)>
+%c0 = arith.constant 2 : index
+%0 = affine.apply #map()[%c0]
```

Comment:
Can you please instead add this as a test case to `tests/filecheck/dialects/affine/affine_ops.mlir` and `tests/filecheck/mlir-conversion/with-mlir/dialects/affine/affine_ops.mlir`?

Reply:
ah thanks, those were the tests I was looking for :)